sBTC SIP

```diff
@@ -0,0 +1,166 @@
+
+sBTC aims to solve these problems by enabling secure movement of BTC in and out of the Stacks network, providing fully decentralized and scalable Bitcoin applications. The goal is to implement a protocol that allows BTC holders to participate in smart contracts without relying on trusted third parties. By addressing these challenges, sBTC seeks to create new opportunities for Bitcoin, enhancing its utility and accessibility while preserving the core principles of decentralization and self-sovereignty.
+
+## Proposed Solution
+The sBTC protocol enables a secure transfer of Bitcoin to and from the Stacks blockchain, providing users with a digital asset backed 1:1 with BTC that can utilize the scalability and programmability of Stacks. Users can deposit BTC into the protocol and seamlessly transact using smart contracts on the Stacks blockchain, and have the freedom to redeem sBTC tokens for the underlying BTC at any time.
+
+The Stacks blockchain allows for onchain verification of the BTC held in the protocol, enhancing transparency compared to custodial approaches like WBTC. Additionally, the management of the Bitcoin script/wallet on the Bitcoin blockchain is decentralized, involving multiple participants rather than a single entity. This ensures a more resilient and trustworthy system, where signers are economically incentivized to execute peg-out transactions efficiently.
+
+Overall, the sBTC protocol not only addresses the limitations of the Bitcoin scripting system but also provides a secure and decentralized solution for utilizing Bitcoin in various applications.
+
+## Design
+To achieve these goals, this proposal implements the following features for sBTC:
+- sBTC is a SIP-10 token backed 1-1 by BTC.
+- The sBTC peg wallet is maintained by the set of sBTC signers. These signers are responsible for the security and maintenance of the wallet, ensuring that sBTC is redeemable for BTC.
+- To support deposits and withdrawals in sBTC, Signers must collectively make smart contract calls on Stacks. We require each signer to individually sign any contract call and ensure at least 70% of the signer signatures are present for each contract call to be valid.
+- Bitcoin can be converted into sBTC within 3 Bitcoin blocks. sBTC can be converted into Bitcoin within 6 Bitcoin blocks. This ensures sBTC on and off ramps are faster than other BTC assets on the market.
+- SIP-10 token contact remains consistent across sBTC releases. This provides reliability for users and developers, meaning no adjustment from builders will be needed as the system evolves through subsequent releases.
+
+### Iterating Toward an Open Signer Network
+This SIP describes an iterative approach to sBTC development via a release plan that is simpler to implement and helps accelerate the sBTC release timeline, but does not include the complete feature set described in the original sBTC design documents.
```

Comment:
This needs to be conveyed in the abstract

Reply:
clarification please:
`Open Signer Network`
and
`sBTC release`

Reply:
Is there a link that can be added for "the original sBTC design documents" ?